docs/source/en/main_classes/deepspeed.mdx

```diff
@@ -1700,7 +1700,7 @@ checkpoint), then you can finish the training by first saving the final model ex
 from deepspeed.utils.zero_to_fp32 import load_state_dict_from_zero_checkpoint
 
 checkpoint_dir = os.path.join(trainer.args.output_dir, "checkpoint-final")
-trainer.deepspeed.save_checkpoint(checkpoint_dir)
+trainer.model_wrapped.save_checkpoint(checkpoint_dir)
```

Comment:
Does this mean that this change will break BC? 

I thought breaking BC was definitely not allowed.

Reply:
but this one is probably super-minor - I can't guess but my hunch is that probably barely anybody is using this code.